src/User/Infra/Form/Type/HashedPasswordType.php

```diff
@@ -64,6 +64,11 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
 
                     $valid = $this->passwordHashing->isValid($user->getPassword(), $plainPassword, $algorithm);
                 }
+                unset($plainPassword); // reference
```

Comment:
You need to unset _after_ the sodium_memzero() otherwise their is no reference anymore 😛

Reply:
nice catch :D fixed.